### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by NacymeG**

### CODE REVIEW

**Review by Caroline-CD — DISMISSED** [review]
OK for me :paintbrush:

File: src/Pages/Signup/style.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    text-align: start;
+  }
+`;
+const Input = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 5px 0;
+  input {
+    /* margin: 5px 0 0 0; */
```
Comment (Hacene010) [inline src/Pages/Signup/style.jsx]: dead code here !!!

**Review by Hacene010 — CHANGES_REQUESTED** [review]
One small evil

**Review by Caroline-CD — APPROVED** [review]
OK :100: for me

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**